docs/architecture/discovery/pepr-hld.md

```diff
@@ -8,67 +99,387 @@ This structure contains entities that cover the majority of use cases for 2026 d
+PRN-SIGNATORY }|--|{ PRN : authorises
+APPROVER }|--|{ WASTE-RECORD : approves
+REGULATOR }|--|{ REPORT : receives
+REGULATOR }|--|{ PRN : cancels
+USER }|--|{ REPORT : creates
+APPROVER }|--|{ REPORT : submits
 
 %% Effects
-summaryLog ||--|| wasteRecord : creates
-summaryLog ||--o| wasteRecord : adjusts
-wasteRecord }|--|| wasteBalance : updates
-prn }|--|| wasteBalance : updates
+SUMMARY-LOG-FILE ||..|| SUMMARY-LOG : "creates (does not create Waste Record)"
+SUMMARY-LOG ||--|| WASTE-RECORD : creates
+SUMMARY-LOG ||--o| WASTE-RECORD : adjusts
+WASTE-RECORD }|--|| WASTE-BALANCE : updates
+PRN }|--|| WASTE-BALANCE : updates
 ```
 
-## Specific structures & use cases
+## Who is using this service
 
-### Exporters
+### Producers
 
-TBD
+Producers are organisations who use packaging for products sold to consumers.
+
+This packaging can be recycled for use in a circular economy, which is a sustainable economic model that aims to eliminate waste and pollution.
+
+Producers purchase PRNs/PERNs (Packaging Recycling Notes/Packaging Export Recycling Notes) from Operators to fulfil their obligations to regulators.
+
+> [!INFO]
+> Producers do not have direct access to this service,
+> their access to PRNs is via another service called RPD and there is an integration between these services.
+
+### Operators
+
+Operators are the regulated organisations who handle waste material.
+
+They fall into two categories of activity: reprocessors and exporters.
+
+These activities are not mutually exclusive, operators can perform both activities on the same or different materials.
+
+#### Reprocessors
+
+Reprocessors receive waste material and recycle it into a raw material for use in the economy, e.g. paper waste is recycled into cardboard boxes.
+
+#### Exporters
+
+Exporters export waste material to a third party, e.g. a waste processing plant in another country.
 
 ### Consultants
 
+Consultants act on behalf of multiple operators, they can usually perform the same actions as privileged users.
+
+### Regulators
+
+The regulators are the authorities responsible for the regulation of waste material. There are four nations in the UK that have regulatory bodies:
+
+1. England: Environment Agency
+2. Northern Ireland: Northern Ireland Environment Agency
+3. Scotland: Scottish Environment Protection Agency
+4. Wales: Natural Resources Wales
+
+### Service Maintainers
+
+The Defra team(s) responsible for maintaining the service used by operators, consultants and regulators.
+
+## What can these users do?
+
+The following actions depend on the user's role:
+
+1. Create or update a user
+2. Upload a summary log, which in turn creates or updates a waste record and may update the waste balance
+3. Create or update a PRN
+4. Create or update a report
+
+Note that updating an entity _may_ include changing its `status`. See a summary of entity statuses below, or see the [Low Level Design](pepr-lld.md) for more details.
+
+## What are the statuses of each entity?
```

Comment:
Looks like we lost the statuses of a `Summary Log` entity here - is that intentional